types/index.d.ts

```diff
@@ -289,10 +289,61 @@ declare namespace PhonegapPluginPush {
 		from?: string
 		notId?: string
 	}
-
+  /**
+   * Set Channel Variable,once created cannot be changed programmtically for importance and vibration
+   */
+  interface channel{
+    /**
+     * channel ID
+     */
+    id: string
+    /**
+     * camillebeaumont name and description fix
+     */
+    name: string 
+    /**
+     * camillebeaumont name and description fix,channel description
+     */
+    description: string
+    /**
+     * channel importance level
+     */
+    importance: integer
+    /**
+     * enable vibration for the notification
+     */
+    vibration: boolean
+    /**
+     * Set notitfication visibility 
+     */
+    visibility: integer
+    /**
+     * Set Channel Group
+     */
+    groupId:string
```

Comment:
Are all of the attributes on the `Channel` interface mandatory? Or can the user ommit some of them? If any of them happens to be optional, you can mark it like so:
```suggestion
    groupId?: string
```

Reply:
I just realized we don't have these documented [here](https://github.com/phonegap/phonegap-plugin-push/blob/master/docs/API.md#channel-properties) with regards to:
- are them optional?
- do they have default values?
So I'll take my previous comment back, it's not your responsibility to fix this.

Reply:
groupId is optional

Reply:
So let's have that suggestion back :smile: